profiler/src/ProfilerEngine/Datadog.Profiler.Native/ClrEventsParser.h

```diff
@@ -0,0 +1,149 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2022 Datadog, Inc.
+
+#pragma once
+
+// from dotnet coreclr includes
+#include "cor.h"
+#include "corprof.h"
+// end
+
+#include <string>
+#include <map>
+#include <shared_mutex>
+#include <math.h>
+
+#include "IAllocationsListener.h"
+
+#include "shared/src/native-src/string.h"
+#include "assert.h"
+
+
+#define LONG_LENGTH 1024
+
+// from https://github.com/dotnet/samples/blob/main/core/profiling/eventpipe/src/CorProfiler.h
+template <class Key, class Value>
+class ThreadSafeMap
```

Comment:
maybe put this in its own file

Reply:
taken from MS sample : removing it with the rest of the traces